change/@ni-nimble-components-d0cf5648-9e73-4eca-8266-c062a09c0dab.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
> if you want to customize two different combobox/select/rich-text-listboxes on the same page, they have to be scoped to distinct theme provider elements.

If in integration we are really finding that we need to customize this for every use case then we should probably add an attribute to make that configuration easier. Not something for this PR but a question we should clarify in a follow-up. I'd hope the generic "No items found" is good enough. But if not we should add an attribute instead of many theme-provider + label-providers all over.

Reply:
> Not something for this PR but a question we should clarify in a follow-up.

FYI @fredvisser

Reply:
Good question - Milan and I discussed, and I'll make the usage/pattern guidance consistent with our current capabilities ("No items found"), and change the guidance and our implementation in the future if this ever becomes the most important thing we can do to improve usability.

Reply:
So, to be clear, once this gets pushed to SLE, should I update the [existing, customized "No tags found" message](https://ni.visualstudio.com/DevCentral/_git/Skyline/pullrequest/499882) to use a `nimble-label-provider-core` instead, or should I remove the customization altogether?

Reply:
My understanding of the discussion would be to remove control-specific customizations of the "no items found" message and in the future if it is desired to have customized messages then have usage guidance to clarify if it's only for some cases or the general case and if the general case add an attribute to make config easier.